src/ice_thermo_cpl.F90

```diff
@@ -474,11 +474,6 @@ subroutine ice_surftemp(h,hsn,a2ihf,t)
   real(kind=WP)  zcprosn
   !---- local parameters
   real(kind=WP), parameter :: dice  = 0.05_WP                       ! ECHAM6's thickness for top ice "layer"
-  !---- freezing temperature of sea-water [K]
-  real(kind=WP)  :: TFrezs
-
```

Comment:
TFrezs is still needed in line 480 for computation of

  zicefl=con*TFrezs/zsniced             ! Conductive heat flux through sea ice [W/m²]

so we should keep it?

Reply:
I'm not fully there today it seems. Yes.